README.md

```diff
@@ -51,7 +51,7 @@ for r in circleguard.verify(1699366, 12092800, 7477458, False):
 
 ### More Generally
 
-The much more flexible way to use circleguard is to make your own Check object and run circleguard with that. This allows for mixing different types of Replay objects - comparing local .osr's to online replays - as well as the liberty to instantiate the Replay objects yourself and use your own Replay subclasses. See [Advanced Usage](#subclassing-replay) for more on subclassing.
+The much more flexible way to use circleguard is to make your Check object and run circleguard with that. This allows for mixing different types of Replay objects - comparing local .osr's to online replays - as well as the liberty to instantiate the Replay objects yourself and use your Replay subclasses. See [Advanced Usage](#subclassing-replay) for more on subclassing.
```

Comment:
want to emphasize that the objects are different from ours; we provide Replay subclasses, but they can use their *own* if they so choose.

Reply:
Just seems like "your own" is a double association - but I will change it back.

Reply:
you're correct, but it holds emphasis *because* it's a double association